scripts/build/termux_buffered_output.sh

```diff
@@ -0,0 +1,24 @@
+termux_buffered_output() {
+	( # Do everything in a subshell to avoid any kind of mess.
+		TAG="$1"
+		set +e
```

Comment:
By the way the `set +e` is gonna persist after this function.
Which is probably not what you intended.
So add a `local -`[^1] above it.

[^1]: https://man.archlinux.org/man/bash.1#local

Reply:
It is being done in subshell (effectively child process) so nothing persists there. It should be fine to use it as is.
But I will probably change `func() { (` combo to `func() (`.